Makefile.Integration.mk

```diff
@@ -0,0 +1,18 @@
+.PHONY: docker-images-jaeger-backend
+docker-images-jaeger-backend: PLATFORMS=linux/amd64
+docker-images-jaeger-backend: create-baseimg 
+	for component in "jaeger-agent" "jaeger-collector" "jaeger-query" "jaeger-ingester" "all-in-one" ; do \
+		regex="jaeger-(.*)"; \
+		component_suffix=$$component; \
+		if [[ $$component =~ $$regex ]]; then \
+			component_suffix="$${BASH_REMATCH[1]}"; \
+		fi; \
+		docker buildx build --target $(TARGET) \
+			--tag $(DOCKER_NAMESPACE)/$$component$(SUFFIX):${DOCKER_TAG} \
+			--build-arg base_image=$(BASE_IMAGE) \
+			--build-arg debug_image=$(DEBUG_IMAGE) \
```

Comment:
Should I use the BASE_IMAGE here ?

Reply:
You could try. I see that the way Dockerfile is designed you cannot skip building debug image. But the main time is actually spent on building the base debug image (where we need to download and build delve), so maybe it's ok.